apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -222,14 +210,13 @@ export default class SetupChecklist extends Component {
     }
   }
 
-  surveyLink() {
+  contactSupport() {
     return (
-      <span>
-        <br />
-        Still having trouble? Please{' '}
-        <a href={this.getSurveyURL()}>submit our quick survey</a> about your
-        setup issues.
-      </span>
+      <UnsafeRenderedMarkdown
+        markdown={
+          i18n.stillHavingTrouble() + ' ' + i18n.contactGeneralSupport()
```

Comment:
Will concatenating strings like this work with RTL?

Reply:
What's the reason for using 'UnsafeRenderedMarkdown'?

Reply:
Still looking into this, but in this case, it made more sense to just combine the two messages.

Reply:
Missed the second question earlier. 
contactGeneralSupport() has a link. i.e. `[support@code.org](mailto:support@code.org)`. I checked with Elijah on the best way to include links in i18n strings. Sounds like the best way (if it's a static link) is to use UnsafeRenderedMarkdown. We know this is safe, but we also want to keep track of where our markdown is and parse it in a standard way.

Reply:
Great! Thanks for following up on that!